### ISSUE TITLE
Niepoprawny kod?

### ISSUE THREAD

**Issue by PaulMatuszny** [korona-science/homework-PaulMatuszny#1]
Dlaczego kod do zadania domowego nr1 nie przechodzi żadnych testów pomimo poprawności?

**Comment by Torisels** [comment]
Tak, kod jest niepoprawny. Dane testowe nie ograniczają się do daty po 3 marca.

**Comment by RouNNdeL** [comment]
Nie przechodzą testy zadania nr 3

**Comment by Torisels** [comment]
Proponuję użyć biblioteki do dat. W takim rozwiązaniu trudno rozpatrzeć wszystkie przypadki.
